contracts/exchangeIssuance/ExchangeIssuance.sol

```diff
@@ -0,0 +1,492 @@
+pragma solidity >=0.6.10;
```

Comment:
Please add the Apache Licence at the beginning of the file.

Reply:
Yep - for easy access

```
/*
    Copyright 2021 Index Cooperative
    Licensed under the Apache License, Version 2.0 (the "License");
    you may not use this file except in compliance with the License.
    You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0
    Unless required by applicable law or agreed to in writing, software
    distributed under the License is distributed on an "AS IS" BASIS,
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
    SPDX-License-Identifier: Apache License, Version 2.0
*/
```